flexmeasures/api/v3_0/tests/test_sensor_schedules_fresh_db.py

```diff
@@ -415,7 +415,7 @@ def test_multiple_contracts(
         "consumption-price": {"sensor": price_sensor_id},
         "production-price": {"sensor": price_sensor_id},
         "site-power-capacity": "2 MW",  # should be big enough to avoid any infeasibilities
-        "site-consumption-capacity": "1 kW",  # we'll need to breach this to reach the target
+        "site-consumption-capacity": "0.25 kW",  # we'll need to breach this to reach the target (25 - 12.1 kWh) / 47 hours
```

Comment:
It's really helpful to add the explanation!

However, I don't know where these numbers are coming from (25, 12.1, 47). I believe they can be found in `message_for_trigger_schedule()` and the first two denote `target_value` and `soc_at_start`. And 47h are due to the use of `use_time_window` where that is the length. Correct?